tests/actions.go

```diff
@@ -1593,7 +1602,7 @@ func releaseIsNotFound(err error) bool {
 }
 
 func (oa *operatorActions) cloneOperatorRepo() error {
-	cmd := fmt.Sprintf("git clone https://github.com/pingcap/tidb-operator.git %s", oa.cfg.OperatorRepoDir)
+	cmd := fmt.Sprintf("git clone %s %s", oa.cfg.OperatorRepoUrl, oa.cfg.OperatorRepoDir)
```

Comment:
If the developers modify the chart in their development branch, they have to configure the git url in order to test the chart modification